solo/methods/base.py

```diff
@@ -378,7 +375,7 @@ def num_training_steps(self) -> int:
             dataset_size = self.trainer.limit_train_batches * dataset_size
 
             num_devices = self.trainer.num_devices
-            num_nodes = self.extra_args.get("num_nodes_horovod", 0) or self.trainer.num_nodes or 1
+            num_nodes = self.trainer.num_nodes or 1
```

Comment:
`self.trainer.num_nodes` can be none?

Reply:
I hope not :) we can maybe trust that it isn't

Reply:
yeah